exosphere-shared/example-apps/docker-service/local-service/features/create-many.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Creating multiple tests
```

Comment:
Please slim this service down to the simplest possible service. Only one functionality, and only one test for it. It could just be a ping service, which replies "pong" when receiving "ping".